nilearn/connectome/connectivity_matrices.py

```diff
@@ -357,12 +436,14 @@ def transform(self, X):
 
         Parameters
         ----------
-        X : list of numpy.ndarray with shapes (n_samples, n_features)
+        X : list of n_subjects numpy.ndarray with shapes \
+            (n_samples, n_features)
             The input subjects time series.
 
+
         Returns
         -------
-        output : numpy.ndarray, shape (n_samples, n_features, n_features)
+        output : numpy.ndarray, shape (n_subjects, n_features, n_features)
```

Comment:
because there is the number of the timeseries scans, which is n_samples, and the number of subjects